yt_dlp/extractor/espn.py

```diff
@@ -73,15 +81,6 @@ class ESPNIE(OnceIE):
     }, {
         'url': 'https://cdn.espn.go.com/video/clip/_/id/19771774',
         'only_matching': True,
-    }, {
-        'url': 'http://www.espn.com/watch/player?id=19141491',
-        'only_matching': True,
-    }, {
-        'url': 'http://www.espn.com/watch/player?bucketId=257&id=19505875',
-        'only_matching': True,
-    }, {
-        'url': 'http://www.espn.com/watch/player/_/id/19141491',
-        'only_matching': True,
```

Comment:
Add these to the new extractor

Reply:
Odd one here, I was unaware those URLs are still active, and they definitely don't fit the WatchESPN syntax and formatting. Although, it appears they still match the original ESPNIE, so I can add them back in there.

Reply:
add them back then